cromulent/vocab.py

```diff
@@ -582,7 +582,7 @@ def rights_getter(self, what):
 		elif what == "part_of":
 			return self.c_part_of
 		else:
-			getattr(self, what)
+			object.__getattr__(self, what)
```

Comment:
Why is this change needed?

Reply:
It works as expected in python2, but in python3 it fails with an infinite recursion error, when the object is being constructed from the reader. I don't know why exactly!